### PR TITLE
LUTECE-2197 : Allow to see the current loggers in the system

### PR DESCRIPTION
**Pull request by jonenst**
We should discuss where we want this feature. Also, do we want a general html editor for logger configuration, or just a list of the runtime loggers and configure everything through the properties file.
Also maybe make the list nicer : colors, collapsing hiearchies, filters, searchs, etc?

### CODE REVIEW

File: webapp/WEB-INF/templates/admin/system/manage_files_system.html [hunk 1]
```diff
@@ -21,5 +21,22 @@
...
+			<@boxBody>
+				<@table>
+					<tr>
+						<th>#i18n{portal.system.manage_files_system.labelLoggerLevel}</th>
+						<th>#i18n{portal.system.manage_files_system.labelLoggerName}</th>
+					</tr>
+					<#list loggers_list as logger >
+					<tr>
+						<td>${logger.effectiveLevel}</td>
+						<td>${logger.name}</td>
```
Comment (rzara) [inline webapp/WEB-INF/templates/admin/system/manage_files_system.html]: You should html escape these

**Comment by rzara** [comment]
Maybe switch to log4j2 which support JMX :-) You get a console which can change log levels "for free"

File: webapp/WEB-INF/templates/admin/system/manage_files_system.html (hunk 1 above)
Comment (jonenst) [reply webapp/WEB-INF/templates/admin/system/manage_files_system.html]: yes thanks

**Comment by jonenst** [comment]
Switching to a maintained log system is tracked at https://dev.lutece.paris.fr/jira/browse/LUTECE-2065
I looked into it a bit today and it seems feasible.

**Pull request closed without merging**